src/MetaMorpho.sol

```diff
@@ -787,6 +803,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
     /// @dev Returns the withdrawable amount of assets from the market defined by `marketParams`.
     /// @dev Assumes that the inputs `marketParams` and `id` match.
+    /// @dev Warning: Can revert.
```

Comment:
If we write this in internal functions, shouldn't we also write it for all functions that call them?

Reply:
We can remove it